docker-images/kafka/kafka-thirdparty-libs/cc/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <parent>
+        <groupId>io.strimzi</groupId>
+        <artifactId>strimzi</artifactId>
+        <version>0.16.0-SNAPSHOT</version>
+    </parent>
+    <groupId>io.strimzi</groupId>
+    <version>1.0.0-SNAPSHOT</version>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>kafka-thirdparty-libs</artifactId>
+    <packaging>pom</packaging>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <repositories>
+      <repository>
+        <id>jcenter</id>
+          <url>https://jcenter.bintray.com/</url>
```

Comment:
You were removing this, right?

Reply:
I had this duplicated in docker-images/kafka/kafka-thirdparty-libs/2.3.x/pom.xml but removed it. It's needed here to pull the Cruise Control jars and dependencies